soc/riscv/riscv-privilege/andes_v5/l2_cache.c

```diff
@@ -100,12 +103,16 @@ static int andes_v5_l2c_init(const struct device *dev)
 			/* This SoC doesn't have L2 cache */
 			return -ENODEV;
 		}
+	} else {
+		LOG_DBG("Init might fail in some hardware combinations "
+					"if syscon driver isn't ready\n");
```

Comment:
nit: `Init might fail for some hardware combinations ....`

Reply:
Thanks, fixed it.